cirq_superstaq/ops/qubit_gates.py

```diff
@@ -711,23 +759,15 @@ def _from_json_dict_(cls, **kwargs: Any) -> "IXGate":
 
 
 def custom_resolver(cirq_type: str) -> Union[Type[cirq.Gate], None]:
-    if cirq_type == "ZZSwapGate":
-        return ZZSwapGate
-    if cirq_type == "Barrier":
-        return Barrier
-    if cirq_type == "ZXPowGate":
-        return ZXPowGate
-    if cirq_type == "AceCR":
-        return AceCR
-    if cirq_type == "ParallelGates":
-        return ParallelGates
-    if cirq_type == "MSGate":
-        return cirq.ops.MSGate
-    if cirq_type == "RGate":
-        return RGate
-    if cirq_type == "IXGate":
-        return IXGate
-    if cirq_type == "ParallelRGate":
-        return ParallelRGate
-
-    return None
+    TYPE_TO_GATE_MAP: Dict[str, Type[cirq.Gate]] = {
```

Comment:
nice! though imo this shouldn't be capitalized if it's not global
```suggestion
    type_to_gate_map: Dict[str, Type[cirq.Gate]] = {
```